docs/contents/releases/1-19/1/changelog.md

```diff
@@ -56,7 +56,7 @@ application of the patches.
 |-----------------------|-------------------|
```

Comment:
nit: all-caps filename `CHANGELOG.md` to go by convention. Also need to change the link in below file if this change goes in.

Reply:
Done. Need to look at other comment though.